doc/sphinxext/gen_rst.py

```diff
@@ -67,12 +67,11 @@ def execfile(filename, global_vars=None, local_vars=None):
     import matplotlib
```

Comment:
we'll switch to https://github.com/sphinx-gallery/sphinx-gallery soon so our changes should land there. It's better for the long run.